pkgs/tools/networking/termscp/default.nix

```diff
@@ -1,61 +1,69 @@
-{ lib
-, stdenv
-, dbus
-, fetchFromGitHub
-, openssl
-, pkg-config
-, rustPlatform
-, AppKit
-, Cocoa
-, Foundation
-, Security
-, samba
+{
+  lib,
+  stdenv,
+  dbus,
+  fetchFromGitHub,
+  openssl,
+  pkg-config,
+  rustPlatform,
+  AppKit,
+  Cocoa,
+  Foundation,
+  Security,
+  samba,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "termscp";
-  version = "0.15.0";
+  version = "0.16.1";
 
   src = fetchFromGitHub {
     owner = "veeso";
     repo = "termscp";
-    rev = "refs/tags/v${version}";
-    hash = "sha256-/WYhwt/GAULX/UY1GyqzauaMRlVuvAwwS0DNfYB7aD4=";
+    rev = "f2efb25ad7c4de35d10a64130e9ac54b1187ff4b";
```

Comment:
A commit message with 1.16.1 was pushed (https://github.com/veeso/termscp/commit/f2efb25ad7c4de35d10a64130e9ac54b1187ff4b) but there was no git tag created for it